src/Payloads/FileContentsPayload.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Spatie\Ray\Payloads;
+
+class FileContentsPayload extends Payload
+{
+    protected string $filename;
+
+    public function __construct(string $filename)
+    {
+        $this->filename = $filename;
+    }
+
+    public function getType(): string
+    {
+        return 'custom';
+    }
+
+    public function getContent(): array
+    {
+        if (!file_exists($this->filename)) {
```

Comment:
Could you rewrite this without the `else` statement? This branch could just return an array with the properties that should be set when the file does not exists. I also don't mind this case using the label `File`.

Could you also add this method to the framework agnostic usage docs?

Reply:
When you say "this case using the label "`File`", do you mean only the case where the file does not exist? or when it does exist - or both?

I assume you mean adding `file()` to the docs - I already did, see the _"Working with files"_ section I added above.  I didn't add a screenshot as the other screenshots are from MacOS and I'm running Linux, so it'd have to be redone anyway.

Reply:
Requested changes submitted.